README.md

```diff
@@ -14,13 +14,16 @@ UI for navigating and organizing OpenAI's ChatGPT conversations.
 
 ![Screenshot](static/screenshot.png)
 
+[Docker Hub Image](https://hub.docker.com/layers/abalmahfadi/chat-history-app/1/images/sha256:5048890aa4a2ab8bd935a59fa12df64b7c028948d3ccd1610b35feedd77fde17?uuid=3C4FE425-AC33-44B0-8EE4-A1C54D21A883)
```

Comment:
what's the purpose of your own prebuilt image that can potentially be tampered, if it can easily be built from scratch?

Reply:
in this section I will remove it from this pull request. But I hope you didn't mind to keep it in my own version.

If you have any other comments on this step I would be happy to hear them.

Thanks